pkg/kubecost/allocationmatcher.go

```diff
@@ -256,5 +258,17 @@ func convertAliasFilterToLabelAnnotationFilter(aliasKey string, filterValue stri
 				annotationOp,
 			),
 		),
-	), nil
+	)
+	var node ast.FilterNode
+	if filterValue == "" || filterValue == "__unallocated__" {
+		node = ops.Or(
```

Comment:
Can we add a brief comment here about the non-extant case? Something that describes why we need these specific filter nodes for this specific condition? (It's described well in the PR description, but future authors may appreciate a brief comment in the code.)

Reply:
Added, I'd meant to do that, woops.

Reply:
Excellent, looks great!